features/geometry.sweptdisksolidpolygondal.feature

```diff
@@ -0,0 +1,18 @@
+@informal-proposition
+Feature: SweptDiskSolidPolygonal
+
+  Scenario: IfcSweptDiskSolidPolygonal
+
+      Given An IfcSweptDiskSolidPolygonal
+        And FilletRadius = not null
+        And Directrix forms an open curve
+        
+       Then FilletRadius has to be smaller than or equal to the length of the start and end segments of the Directrix
+        And FilletRadius has to be smaller than or equal to the length / 2 of the inner segments of the Directrix
+
+      Given An IfcSweptDiskSolidPolygonal
+        And FilletRadius = not null
+        And Directrix forms a closed curve
+        
+       Then FilletRadius has to be smaller than or equal to the length of all segments of the Directrix
```

Comment:
How come not `length / 2`, similarly to the open curve example?

Reply:
@pjanck I think you're correct. A closed curve means all segments are the "inner segments" and hence the length / 2 rule should apply. Thank you.

There are also no test cases yet for closed curve directrices.